src/H5FDvfd_swmr.c

```diff
@@ -269,14 +263,14 @@ static herr_t
 H5FD__swmr_reader_open(H5FD_vfd_swmr_t *file)
 {
     h5_retry_t retry;
-    bool do_try;                             /* more tries remain */
+    hbool_t do_try;                         /* more tries remain */
     herr_t      ret_value = SUCCEED;
     FUNC_ENTER_STATIC
 
     file->api_elapsed_nbuckets = file->config.max_lag + 1;
 
     file->api_elapsed_ticks =
-        calloc(file->api_elapsed_nbuckets, sizeof(*file->api_elapsed_ticks));
+        H5MM_calloc(file->api_elapsed_nbuckets * sizeof(*file->api_elapsed_ticks));
 
```

Comment:
Not all memory allocation has been moved to H5MM. Doing so exposes a memory issue that I haven't corrected yet. This causes failures in debug builds when the H5MM sanity checks complain.